### PR TITLE
Rebased: add agents and haproxy_lbaasv2 driver to support lbaasv2

### PR DESCRIPTION
**Pull request by jiahaoliang**
A new PR for lbaasv2 is created after rebase.

Passed pep8.

All comments marked as "Fixed in the rebased PR" in the old PR were fixed.

Old PR link: [https://github.com/oneconvergence/group-based-policy/pull/358](https://github.com/oneconvergence/group-based-policy/pull/358)

### CODE REVIEW

File: gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py [hunk 1]
```diff
@@ -55,7 +55,8 @@
 cfg.CONF.register_opts(service_chain_opts, "simplechain")
 
 # Service chain API supported Values
-sc_supported_type = [pconst.LOADBALANCER, pconst.FIREWALL]
```
Comment (njagadish) [inline gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py:58]: We can go ahead and merge the following files into One Convergence NFP's GBP code. However, we need to submit to GBP.
 gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py 
 gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/heat_node_driver.py

File: gbpservice/nfp/config_orchestrator/common/lbv2_constants.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```
Comment (njagadish) [inline gbpservice/nfp/config_orchestrator/common/lbv2_constants.py:1]: I don't think, this file can go into common directory. Can we move these to LBV2 specific file.

**Comment by ashutosh-mishra** [comment]
@jiahaoliang - Why we are maintaining octavia code in configurator ?
If we are using that as a library, Can we keep only required files or install it rather then maintain explicitly.

**Comment by njagadish** [comment]
This code is merge-able except for the following. These can be taken up in next task post merge.
(1) Maintaining Octavia code
(2) Few files to be submitted to GBP
(3) Need to tests if existing unit tests are broken because of these changes

**Comment by jiahaoliang** [comment]
@ashutosh-mishra @njagadish Initially we tried to avoid the dependency of installing Octavia. But now we need to install Octavia anyway, I can remove those Octavia lib code and import them from Octavia pip package.

File: gbpservice/nfp/config_orchestrator/common/lbv2_constants.py (hunk 2 above)
Comment (jiahaoliang) [inline gbpservice/nfp/config_orchestrator/common/lbv2_constants.py:1]: @njagadish Those constants only used in notification handler, do you recommend me to move them to the beginning of gbpservice/nfp/config_orchestrator/handlers/notification/handler.py ?

**Pull request merged**